incubator/012-fix-to-fresh-images/init-data

```diff
@@ -0,0 +1,361 @@
+#!/bin/bash
+
+# Copyright 2015--2019 Jan Pazdziora
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Initialization of /data (bind-mounted volume) from /data-template
+# we IPA server was not yet configured.
+
+set -e
+
+# Turn on tracing of this script
+test -z "$DEBUG_TRACE" || {
+	set -x
+	if [ "${DEBUG_TRACE}" == "2" ]; then
+		export SYSTEMD_LOG_LEVEL="debug"
+		export SYSTEMD_LOG_TARGET="journal+console"
+		export SYSTEMD_LOG_COLOR="no"
+		export DEBUG=1
+	fi
+}
+
+cd /
+
+function is_kubernetes
+{
+	# If KUBERNETES is not empty, return true
+	[ -n "${KUBERNETES}" ] && return 0
+
+	# This is true when 'automountServiceAccountToken: true'
+	# into the Pod spec
+	[ -e "/var/run/secrets/kubernetes.io/serviceaccount" ] && return 0
+
+	# Else return false
+	return 1
+}
+
+function set_systemd_units_private_tmp_off
```

Comment:
PrivateTmp=on was evoking some systemd units to fail. However this evoke that the services which specify PrivateTmp=on, are not going to have an isolated /tmp directory. Dig more about this.